index.js

```diff
@@ -131,6 +156,19 @@ class Assets {
       });
   }
 
+  removeBucket(bucket) {
+    console.log(`Removing bucket`, bucket);
+    const deleteParams = {
+      Bucket: bucket,
+    };
+
+    return this.provider.request('S3', 'deleteBucket', deleteParams)
+      .then(() => {
+        this.log(`Did it fail?`)
```

Comment:
What is a good way of checking for failure here, and capturing a 404 instead of reporting it to the stack?